mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+## v1.18.0 (May 13, 2025)
```

Comment:
drop this, we'll add during release.

the important thing is that we add the new features section